default/scripting/techs/spy/CUSTOM_ADVISORIES.focs.txt

```diff
@@ -6,4 +6,28 @@ Tech
     researchcost = 1
     researchturns = 1
     tags = [ "PEDIA_SPY_CATEGORY" ]
+    effectsgroups = [
+        [[CUSTOM_SITREPS]]
+
+        EffectsGroup      // colonies consume influence proportional to how many planets the empire controls
```

Comment:
this does not fit here

Reply:
You mean it makes little sense to have influence upkeep code in CUSTOM_ADVISORIES.focs.txt, right?
I fully agree.
Should I create a new tech to be the root of influence upkeep (and related) stuff?

Reply:
Would be better than having it here.